protobuf-converter/src/test/java/io/confluent/connect/protobuf/ProtobufDataTest.java

```diff
@@ -774,6 +788,103 @@ public void testToConnectSchemaMismatchArray() {
+    assertEquals("test_sfixed64", fieldElem.getName());
+    assertEquals("sfixed64", fieldElem.getType());
+    fieldElem = messageElem.getFields().get(11);
+    assertEquals("test_sint32", fieldElem.getName());
+    assertEquals("sint32", fieldElem.getType());
+    fieldElem = messageElem.getFields().get(12);
+    assertEquals("test_sint64", fieldElem.getName());
+    assertEquals("sint64", fieldElem.getType());
+    fieldElem = messageElem.getFields().get(13);
+    assertEquals("test_uint32", fieldElem.getName());
+    assertEquals("uint32", fieldElem.getType());
+    fieldElem = messageElem.getFields().get(14);
+    assertEquals("test_uint64", fieldElem.getName());
+    assertEquals("uint64", fieldElem.getType());
+
+    Descriptors.FieldDescriptor fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_fixed32");
+    assertEquals(12, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_fixed64");
+    assertEquals(12L, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_sfixed32");
+    assertEquals(12, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_sfixed64");
+    assertEquals(12L, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_sint32");
+    assertEquals(12, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_sint64");
+    assertEquals(12L, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_uint32");
+    assertEquals(12, message.getField(fieldDescriptor));
+    fieldDescriptor = message.getDescriptorForType()
+        .findFieldByName("test_uint64");
+    assertEquals(12L, message.getField(fieldDescriptor));
+
+    TestMessageProtos.TestMessage convertedMessage = TestMessageProtos.TestMessage.parseFrom(message.toByteArray());
+    assertEquals(originalMessage, convertedMessage);
+  }
+
+  @Test
+  public void testRoundTripConnectUInt32Fixed32() throws Exception {
+    final Long UNSIGNED_RESULT = 4294967295L;
+    Integer expectedValue = -1;
+
+    TestMessageProtos.TestMessage message = TestMessageProtos.TestMessage
+        .newBuilder()
+        .setTestFixed32(expectedValue)
+        .setTestUint32(expectedValue)
+        .build();
+    SchemaAndValue result = getSchemaAndValue(message);
+
+    ProtobufData protobufData = new ProtobufData();
+    ProtobufSchemaAndValue converted = protobufData.fromConnectData(result.schema(), result.value());
+    DynamicMessage convertedValue = (DynamicMessage) converted.getValue();
+
+    TestMessageProtos.TestMessage parsedMessage = TestMessageProtos.TestMessage.parseFrom(convertedValue.toByteArray());
+//    System.out.println(parsedMessage);
```

Comment:
nit: remove this line and the next line

Reply:
So are these two asserts correct? 
```    
assertEquals(message, parsedMessage);
assertTrue(parsedMessage.toString().contains(String.valueOf(UNSIGNED_RESULT)));
```

FYI, `System.out.println(parsedMessage);` would show 4294967295
but `System.out.println(parsedMessage.getTestFixed32())` still show -1. (My understanding is that's due to the binary encoding and decoding)